## Contact
---
Email: tmajarian@gmail.com  
Linkedin: [https://www.linkedin.com/in/tmajarian/](https://www.linkedin.com/in/tmajarian/)  

## Experience
---
**The Broad Institute**  
*Computational Associate II*  
July 2020 – Present  
Led effort in developing and testing systems for interoperating among multiple NIH-backed cloud platforms. Coordinated grant submission, managed costs and progress, completed monthly deliverables for an NIH subcontract award. Created and maintained multiple large-scale biobank resources for multiple labs across Broad and Mass. General Hospital. Created and implemented various polygenic scores for metabolomic traits and diseases. Mentored and managed multiple junior scientists and trainees.  

*Computational Associate I*  
August 2017 - July 2020  
Developed cloud computing pipelines for genetic analyses including GWAS, LD score regression, credible set analysis. Led multiple training sessions on cloud-based biomedical research attended by over 100 scientists. Contributed to multiple international research consortia focused on type 2 diabetes and related clinical phenotypes. Developed novel methods for assessing the impact of rare genetic variation on complex human disease and predicting target genes from GWAS loci.  

## Education
---
**Carnegie Mellon University**  
2015-2017  
MS in Computational Biology  
Distinguished Research Award  
Coursework: Computational genomics, Machine learning, Biological modeling and simulation, Automation in biological research, Bioimage informatics

**The University of Vermont**  
2009-2013  
BA in Applied Mathematics & Biology  
Honors College  

## Publications 
---
- DiCorpo, D., Gaynor, S., Russell, E. et al. (2020) Whole Genome Sequence Association Analysis of Fasting Glucose and Fasting Insulin Levels in Diverse Cohorts from the NHLBI TOPMed Program. medrxiv. doi.org/10.1101/2020.12.31.20234310. 
- Wessel, J., Majarian, T., Highland, H. et al. (2020) Rare Non-coding Variation Identified by Large Scale Whole Genome Sequencing Reveals Unexplained Heritability of Type 2 Diabetes. medrxiv. doi.org/10.1101/2020.11.13.20221812  
- Laville, V., Majarian, T., Sung, Y. J., et al. (2020) Large-scale multivariate multi-ancestry Interaction analyses point towards different genetic mechanisms by population and exposure. biorxiv. doi.org/10.1101/562157  
- Laville, V., Majarian, T., de Vries, P.S. et al. (2020) Deriving stratified effects from joint models investigating gene-environment interactions. BMC Bioinformatics, 21, 251. doi.org/10.1186/s12859-020-03569-4  
- Majarian, T., Murphy, R.F. and Lakdawala, S. (2019) Learning the sequence of influenza A genome assembly during viral replication using point process models and fluorescence in situ hybridization. PLoS Comp Bio. doi.org/10.1371/journal.pcbi.1006199  
- Majarian T., Cao-Berg I., Ruan X. and Murphy R. F. (2019) CellOrganizer: Learning and Using Cell Geometries for Spatial Cell Simulations. In: Hlavacek W. (eds) Modeling Biomolecular Site Dynamics. Methods in Molecular Biology, vol 1945. Humana Press, New York, NY. doi.org/10.1007/978-1-4939-9102-0_11  
- Johnson, G. R., Kangas, J. D., Dovzhenko, A., Trojok, R., Voigt, K., Majarian, T., Palme, K. and Murphy, R. F. (2017), A method for characterizing phenotypic changes in highly variable cell populations and its application to high content screening of Arabidopsis
thaliana protoplasts. Cytometry, 91: 326–335. doi.org/10.1002/cyto.a.23067  
- Li, Y., Majarian, T., Naik, A. W., Johnson, G. R. and Murphy, R. F. (2016), Point process models for localization and interdependence of punctate cellular structures. Cytometry, 89: 633– 643. doi.org/10.1002/cyto.a.22873  

## Presentations 
---
- *NIH Cloud Platform Interoperability Effort (NCPI) Spring Workshop*, 2021  
Proof of concept of interoperable approaches for improving outcomes of pediatric diseases [(recording available at timestamp 13:45)](https://cbiit.webex.com/recordingservice/sites/cbiit/recording/e403162f8f171039adbf0050568cfa13/playback)  
- *The American Society for Human Genetics meeting*, 2019  
Poster: Colocalizing gene-lifestyle interaction associations with molecular signatures of smoking and alcohol consumption  
Workshop instructor: Reproducible Analysis on the Cloud: Creating and Sharing Methods and Analyses with Terra [(workshop material)](https://app.terra.bio/#workspaces/amp-t2d-op/2019_ASHG_Reproducible_GWAS-V2)  
- *The Cohorts for Heart and Aging Research in Genomic Epidemiology Consortium meeting*, St. Louis MI, 2019  
Oral presentation: Colocalizing gene-lifestyle interaction associations with molecular signatures of smoking and alcohol consumption
- *University of Washington Summer Institute in Statistical Genetics*, 2019  
Instructor: Computational pipelines for whole genome sequence data
- *World Preclinical Congress*, Boston MA, 2018  
Oral presentation: Machine Learning of the Assembly Instructions of a Cell
- *American Society of Human Genetics*, San Diego CA, 2018  
Poster presentation: Leveraging T2D specific omics data in rare variant association analysis in TOPMed
- *National Center for Multiscale Modeling of Biological Systems Cell Modeling workshop*, Pittsburgh PA, 2016  
Oral presentation: Approaches to modeling protein distributions: vesicle modeling in CellOrganizer through Gaussian mixture and point process models

